packages/airnode-deployer/docker/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM node:14.17-alpine3.14
 
 ENV name="airnode-deployer"
-ENV terraformURL="https://releases.hashicorp.com/terraform/1.0.6/terraform_1.0.6_linux_amd64.zip" \
+ENV terraformURL="https://releases.hashicorp.com/terraform/1.0.6/terraform_1.0.6_linux_" \
```

Comment:
Why this change? Is this platform agnostic and the previous was not?

Reply:
Mac OS presents in Alpine docker images on an x86_64 machine as `x86_64`, and on "M1" Macs as `aarch64` - these correspond to Terraform's `linux-amd64` and `linux-arm64` download options and also to Docker's architecture types.
Linux machines behave the same way; a Raspberry Pi running 64 bit Debian during the docker build presents as `aarch64` and `x86_64` on an x86_64 Linux host machine.

Reply:
I see - this should be named `baseTerraformURL` or something since it is an invalid URL on it's own